<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteInstaller</class>
 <widget class="QWidget" name="RemoteInstaller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>422</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="connectionInformation">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Before you use the remote installer please check that:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator_4"/>
   </item>
   <item>
    <widget class="QLabel" name="sshdInformation">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;the sshd daemon has started in the target pc&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;the ssh port is opened&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="sourceTitle">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Source&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KSeparator" name="srcSeparator"/>
   </item>
   <item>
    <widget class="QLabel" name="sourceLabel">
     <property name="text">
      <string>Choose the source directory of your package.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KUrlRequester" name="sourceDirectory"/>
   </item>
   <item>
    <widget class="QLabel" name="username">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator"/>
   </item>
   <item>
    <widget class="QLabel" name="usernameTarget">
     <property name="text">
      <string>Enter the username of the target pc.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" name="usernameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="hostname">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Hostname&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator_2"/>
   </item>
   <item>
    <widget class="QLabel" name="hostnameTarget">
     <property name="text">
      <string>Enter the hostname of the target pc</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KLineEdit" name="hostnameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="installPackageLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Install Package&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KPushButton" name="installButton">
     <property name="text">
      <string>Install the package</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
